docs/Catalog/URI.md

```diff
@@ -0,0 +1,59 @@
+<!-- markdownlint-disable-next-line first-line-h1 -->
+Every package and object in the Bucket and Packages views has a "CODE" pane,
+which contains code snippets that can be used to download and upload a package
+or object via either:
+
+- Python (API) ![Python](../imgs/uri-python.png)
+- CLI (shell commands) ![CLI](../imgs/uri-cli.png)
+
+In addition, Packages have a third tab that returns a Quilt+ URI:
+
+- URI (identifier) ![URI](../imgs/uri-uri.png)
+
+They all have a `copy` button that copies the code to the clipboard.
+
+## Quilt+ URIs
+
+Quilt+ URIs are a way to uniquely identify a package or object in the Quilt
```

Comment:
referencing packages vs objects seems misleading here, bc "objects" usually refer to "s3 objects", and quilt+ uris don't support those -- they support packages and "subpackages" (e.g. subsets of packages, down to individual package entries).

also, "Quilt bucket" seems somewhat vague as well -- these are "s3 buckets"

Reply:
Switched to subpackage